src/skin/frontend/base/default/omise/gateway/css/style.css

```diff
@@ -45,53 +47,53 @@
     background: #4e2e7f;
 }
 .banks-list .logo-scb {
-    background: url('../images/scb.svg') #4e2e7f;
+    background: url('../images/scb.svg') #4e2e7f !important;
```

Comment:
@jonrandy generally I think we should avoid using `!important` when it is not necessary. 
Some approaches like
```css
.banks-list .omise-logo-wrapper > div.logo-scb {
    background: url('../images/scb.svg') #4e2e7f;
}
```
Would give the same result but reserve `!important` for another case, such 3rd-party developer / designer to customize it.

Btw, i think we may not need to have these changes at all. 
Let me explain in the next comment.

Reply:
explained here https://github.com/omise/omise-magento/pull/184#issuecomment-464032191